sdk/schemaregistry/schema-registry-avro/review/schema-registry-avro.api.md

```diff
@@ -8,8 +8,12 @@ import { SchemaRegistry } from '@azure/schema-registry';
 
 // @public
 export class AvroSerializationError extends Error {
```

Comment:
Should we call this AvroSerializerError?

Reply:
@bterlson I renamed `AvroSerializationError` to `AvroSerializerError` to align with the serializer's class name. What do you think?

Reply:
@bterlson I agree. To clarify, it represents all errors thrown from the class except for `RestError`s coming from the service.

Reply:
For .NET we are going to go with SchemaRegistryAvroException.

/cc @tg-msft

Reply:
Discussed offline, JS will go with `AvroError`.